### PR TITLE
{vis}[GCCcore/7.3.0,intel/2018b] VMD v1.9.3, FLTK v1.3.4, ACTC v1.1, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#7236~~ (FFmpeg)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2008.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d9ceedee32ffbb60ede62338ae4a2453 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3827cd59a333367ffed5ce718fd5005b for a full test report.

File: easybuild/easyconfigs/v/VMD/VMD-1.9.3-intel-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+homepage = 'http://www.ks.uiuc.edu/Research/vmd'
+description = """VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular
+    systems using 3-D graphics and built-in scripting."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+# downloading requires registration
+sources = ['vmd-%(version)s.src.tar.gz']
+patches = ['VMD-%(version)s_configure.patch']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/v/VMD/VMD-1.9.3-intel-2018b-Python-2.7.15.eb]: missing (patch) checksum

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/4388c9b73edae850da60f92f60e35435 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**